Sources/PackageDescription/SupportedPlatforms.swift

```diff
@@ -31,6 +31,9 @@ public struct Platform: Encodable, Equatable {
     /// The watchOS platform.
     public static let watchOS: Platform = Platform(name: "watchos")
 
+    /// The DriverKit platform
+    public static let driverKit: Platform = Platform(name: "driverkit")
```

Comment:
Not sure how to deal with availability here  since it seems like `if #available` does not work for `_PackageDescription`. That means if I add availability here, the static functions can't be compiled.

Reply:
I just recently found out that `_PackageDescription` is a special thing — is it a bug that `#ifavailable` doesn't work for it, do you think?

Reply:
Yah, that's what I was thinking. It seems we so far only added platform constants for platforms without versioning, so we may not have noticed that something is missing here.

Reply:
I think it's not too bad to leave this without availability for now, but we will have to figure something out before we ship another release in the future.